get-lm-desk.sh

```diff
@@ -652,12 +652,21 @@ function install_beeai {
     bold green "INSTALLING BEEAI"
     green "$(term_bar -)"
 
-    # TODO: Guard for missing brew
-    run $brew_bin install i-am-bee/beeai/beeai
-    beeai_bin=$(find_cmd_bin beeai)
-
-    # TODO: Make this non-interactive if requested
-    run $beeai_bin env setup
+    # If brew is available use brew
+    if [ "$brew_bin" != "" ]
+    then
+        run $brew_bin install i-am-bee/beeai/beeai
+        beeai_bin=$(find_cmd_bin beeai)
+
+        # run $beeai_bin env setup
+        echo "BeeAI: configure your preferred LLM provider"
+        run $beeai_bin env add LLM_MODEL=granite-code:3b
```

Comment:
We definitely don't want to use `granite-code:3b`!!

```suggestion
        run $beeai_bin env add LLM_MODEL=granite3.3
```